src/Appwrite/Platform/Modules/Databases/Http/Databases/Collections/Logs/XList.php

```
@@ -71,12 +70,12 @@ public function __construct()
             ->inject('response')
             ->inject('dbForProject')
             ->inject('locale')
-            ->inject('geodb')
+            ->inject('geoRecord')
             ->inject('authorization')
             ->callback($this->action(...));
     }
 
-    public function action(string $databaseId, string $collectionId, array $queries, UtopiaResponse $response, Database $dbForProject, Locale $locale, Reader $geodb, Authorization $authorization): void
+    public function action(string $databaseId, string $collectionId, array $queries, UtopiaResponse $response, Database $dbForProject, Locale $locale, array $geoRecord, Authorization $authorization): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$locale'. (undefined)

(UnusedFormalParameter)